saleor/payment/gateways/stripe/stripe_api.py

```diff
@@ -178,11 +178,18 @@ def create_payment_intent(
         return None, error
 
 
-def update_payment_method(api_key: str, payment_method_id: str, channel_slug: str):
+def update_payment_method(
+    api_key: str,
+    payment_method_id: str,
+    channel_slug: str,
+    metadata: Dict[str, str] = None,
+):
     with stripe_opentracing_trace("stripe.PaymentMethod.modify"):
         try:
             stripe.PaymentMethod.modify(
-                payment_method_id, api_key=api_key, metadata={"channel": channel_slug}
+                payment_method_id,
+                api_key=api_key,
+                metadata={**(metadata or {}), "channel": channel_slug},
```

Comment:
You can drop a channel_slug from here. It is previous approach wich wasn't very well. If storefront would like to store channel slug, then it will pass it as a metadata in checkoutPaymentCreate

Reply:
I think that you can also simplify this by: `metadata=metadata`. We can call this function only when we have some metadata in payment object.